### PR TITLE
feat!: upgrade appVersion to 5.0.1

### PR DESCRIPTION
**Pull request by juanpicado**
# Changes

- Update base image from v4 to v5

## Migration guide

⚠️ ⚠️ ⚠️ Breaking changes, please read the document.

https://verdaccio.org/blog/2021/04/14/verdaccio-5-migration-guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ and their default values.
 | `image.pullPolicy`                 | Image pull policy                                               | `IfNotPresent`                 |
 | `image.pullSecrets`                | Image pull secrets                                              | `[]`                           |
 | `image.repository`                 | Verdaccio container image repository                            | `verdaccio/verdaccio`          |
-| `image.tag`                        | Verdaccio container image tag                                   | `4.12.0`                        |
+| `image.tag`                        | Verdaccio container image tag                                   | `4.12.0`                       |
```
Comment (rblaine95) [inline README.md:89]: The `image.tag` should be `5.0.1`

Comment (juanpicado) [reply README.md:89]: Oh yeah thanks :) I will update

**Review by rblaine95 — DISMISSED** [review]
LGTM, deployed the previous version then upgraded after updating the logging config.
Pod came back up, health checks all happy, can access UI and log in using previously set credentials.

I didn't test publishing a package before upgrading and then checking if the package still exists.

File: README.md (hunk 1 above)
Comment (rblaine95) [reply README.md:89]: The `README.md` should also be updated

Comment (juanpicado) [reply README.md]: done, thanks for the review ! 💯

**Review by rblaine95 — APPROVED** [review]
LGTM

**Review by rmkanda — APPROVED** [review]

File: charts/verdaccio/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-description: A lightweight private npm proxy registry (sinopia fork)
+description: A lightweight private node.js proxy registry
```
Comment (zkochan) [inline charts/verdaccio/Chart.yaml:2]: Is it a proxy though? You can publish to Verdaccio so it is not a proxy, it is just a registry. It is a Node.js registry.

Comment (rblaine95) [reply charts/verdaccio/Chart.yaml:2]: You can also use it as a pull-through cache/proxy of upstream registries

Comment (zkochan) [reply charts/verdaccio/Chart.yaml:2]: I know but that is only one of the many features.

Comment (juanpicado) [reply charts/verdaccio/Chart.yaml:2]: I understand both points, actually, the web even says `npm`. 🤦🏼 for quite a while, anyway at this point everywhere says `private proxy registry`, blog, readmes even talks 😋 . My change was more oriented on remove `npm` and `sinopia` from the sentence, but maybe in the close future, we can rephrase better the description in a better way.

**Comment by juanpicado** [comment]
I'll wait a bit more in case any @verdaccio/collaborators have some extra thoughts on this, even if there is enough approval, the more the better.

**Review by jhonmike — APPROVED** [review]
LGTM

**Pull request merged**